benefit/data/sheets/WorkBasketInputFields/WorkBasketInputFields.json

```diff
@@ -10,12 +10,13 @@
   {"LiveFrom": "18/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "directionDueDate", "Label": "Response due date", "DisplayOrder": 9},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "createdInGapsFrom", "Label": "Is a new digital case?", "DisplayOrder": 10},
   {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "appeal", "ListElementCode": "hearingType", "Label": "Hearing type", "DisplayOrder": 11},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "isScottishCase", "Label": "Is a Scottish case?", "DisplayOrder": 12},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "languagePreferenceWelsh", "Label": "Welsh language flag", "DisplayOrder": 13},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "translationWorkOutstanding", "Label": "Translation outstanding", "DisplayOrder": 14},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "reasonableAdjustmentsOutstanding", "Label": "Alternative letter format required", "DisplayOrder": 15},
-  {"LiveFrom": "20/06/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "hasUnprocessedAudioVideoEvidence", "Label": "Does case have unprocessed audio/video evidence?", "DisplayOrder": 16},
-  {"LiveFrom": "16/12/2021", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpDueDate", "Label": "FTA response due date", "DisplayOrder": 17},
+  {"LiveFrom": "17/08/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "isConfidentialCase", "Label": "Is a Confidential Case?", "DisplayOrder": 12},
```

Comment:
Does this field (and the rest) need to be behind a feature flag?

Reply:
Haven't seen any such requirement

Reply:
Can you enquire about it then to make sure?